test/cli/install/bun-update.test.ts

```diff
@@ -265,7 +265,7 @@ it("should update to latest versions of dependencies", async () => {
     name: "foo",
     dependencies: {
       "@barn/moo": "^0.1.0",
-      baz: "^0.0.5",
+      baz: "~0.0.5",
```

Comment:
Can we run this test for both `~0.0.5` and `^0.0.5`?

Reply:
👍